jupyter-book/introduction/raw_data_processing.md

```diff
@@ -1,52 +1,397 @@
+While this "knee" method is intuitive and can often estimate a reasonable threshold, it has several drawbacks. For example, not all cumulative histograms display an obvious knee, and it is notoriously difficult to design algorithms that can robustly and automatically detect such knees. Finally, the total UMI count associated with a barcode may not, alone, be the best signal to determine if the barcode was associated with an empty or damaged cell.
+
+This led to the development of several tools specifically designed to detect empty or damaged droplets, or cells generally deemed to be of "low quality" {cite}`Lun2019,Heiser2021,Hippen2021,Muskovic2021,Alvarez2020,Young2020`. These tools incorporate a variety of different measures of cell quality, including the frequencies of distinct UMIs, the number of detected genes, and the fraction of mitochondrial RNA, and typically work by applying a statistical model to these features to classify high-quality cells from putative empty droplets or damaged cells. This means that cells can typically be scored, and a final filtering can be selected based on an estimated posterior probability that cells are not empty or compromised. While these models generally work well for single-cell RNA-seq data, one may have to apply several additional filters or heuristics to obtain robust filtering in single-nucleus RNA-seq data{cite}`Kaminow2021,He2022`, like those exposed in the [`empty_drops_cell_ranger`](https://github.com/MarioniLab/DropletUtils/blob/master/R/emptyDropsCellRanger.R) function of DropletUtils{cite}`Lun2019`.
+
+In addition to determining which cell barcodes correspond to empty droplets or damaged cells, one may also wish to identify those cell barcodes that correspond to doublets or multiplets. When a given droplet captures two (doublets) or more (multiplets) cells, this can result in a skewed distribution for these cell barcodes in terms of quantities like the number of reads and UMIs they represent, as well as gene expression profiles they display. Many tools have also been developed to predict the doublet status of cell barcodes{cite}`DePasquale2019,McGinnis2019,Wolock2019,Bais2019,Bernstein2020`. Once detected, cells determined to likely be doublets can be removed or otherwise adjusted for in the subsequent analysis.
+
+% **Should anything else go in this basic QC section?**
```

Comment:
See above I guess. I'd suggest to slightly switch the focus a little bit away from the count matrix to the steps that lead to the count matrix (in terms of QC). The count matrix QC is covered in the single-cell data preprocessing (ambiguous term again, eh?) chapter which includes ambient RNA + doublet QC.

Reply:
We added a new toggle subsection for alevinQC. Feel free to move this part to other chapters.